.github/workflows/continuous-integration.yml

```diff
@@ -23,14 +22,15 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+        dbal-version:
+          - "^2.13"
+          - "^3.1"
```

Comment:
```suggestion
          - "2.13.0"
          - "3.1.0"
```

Reply:
Not sure here. It's locks hard tested versions. `lowest` already check `3.1.0` and ``highest` test any future version.